docs/Grundkonzept.md

```diff
@@ -0,0 +1,28 @@
+
+Sind mehrere Ladepunkte an einer Unterverteilung angeschlossen, deren Zuleitung weniger verträgt, als die Ladepunkte abgeben können, muss man sich noch über das Loadsharing Gedanken machen. Zum Beispiel ist eine Unterverteilung an der zwei 22kW openWBs angeschlossen sind, in den seltensten Fällen mit den dafür notwendigen 63A abgesichert; häufig z.B. nur 35A. Hier fügt man noch eine Komponente "virtuellen Zähler" ein, setzt den im [Lastmanagement](https://github.com/openWB/core/wiki/Lastmanagement-und-kaskadierte-Zähler) in der Struktur über die beiden Ladepunkte und trägt dort die 24kW / 35A ein. Die maximal zulässige Leistung am EVU-Punkt trägt man dementsprechend in diesem [Zähler](https://github.com/openWB/core/wiki/Zaehler) ein.
+ 
+### Fahrzeuge - warum wir das hier alles machen
+Bei den Fahrzeugen zerteilt sich die Konfiguration in die technischen Eigenschaften und die ladungstechnischen Aspekte - das sind die beiden Profile für Fahrzeug und Ladung. Die beiden zusammengefasst ergibt das Fahrzeug. 
+#### Fahrzeug-Profile
+Da steht drin, was für einen Fahrzeugtyp mit welchen Eigenschaften wir haben (ID.3, BMW i3, Tesla Model Y,...) und Angaben zur Ladungssteuerung und -statistiken. Haben wir nur ein Auto oder interessiert Zielladen oder "geladene km" nicht, reicht es beim Standard Fahrzeug-Profil zu bleiben.
+#### Ladeprofil
+Da steht drin, wie wir das Auto geladen werden soll, aber diese Einstellungen sind später im UI der openWB alle änderbar. Bei mehr als einer openWB sollte man je Fahrzeug ein Ladeprofil anzulegen, damit man z.B. von der neu hinzu gekommenenen Priorisierung Gebrauch machen kann. (Das priorisierte Auto fängt zuerst an zu laden, wenn man beide gleich priorisiert, muss der Überschuss wie bei der 1.9 für beide angeschlossenen Autos reichen.) Bei mehr als einem Auto sollte man auch je Auto ein Ladeprofil anlegen, denn da stehen die Zeitpläne und ein SoC-Limit drin. Einfach ist es bei nur einem Auto - da reicht es wieder beim Standard Ladeprofil zu bleiben und das zu ändern. Allerdings wird das Standard Ladeprofil (im Modus Stop) auch dafür genutzt, Ladepunkte nach Abstecken zu sperren. Möchte man also Ladepunkte vor unbefugtem Zugriff schützen, muss man auch bei einer Wallbox ein eigenes Ladeprofil anlegen.
+#### Fahrzeug
+Hier werden nun die beiden Profile zusammengeführt und ggf. ein SoC-Modul konfiguriert.
+
+**Kurz zusammengefasst:** Mit einem Auto und einer openWB bleibt man bei den Profilen immer bei den mitgelieferten Standard-Profilen und ändert die Parameter dort. Bei mehreren Autos legt man für jedes Auto ein Fahrzeug-Profil und ein Ladeprofil an und fasst die beiden im Fahrzeug zusammen.
+
+Credit für dieses Howto geht an [Gero](https://openwb.de/forum/viewtopic.php?t=8076)
```

Comment:
"Credit" bedeutet für mich, dass Du das Copyright von diesem Beitrag inne hast. Das wirft in Zukunft viele Fragen bezüglich Überarbeitung und Lizenzierung etc. auf. Daher würde ich eher folgenden Wortlaut nehmen, wenn das für Dich ok ist.
```suggestion
Danke für das Erstellen dieses Howto an [Gero](https://openwb.de/forum/viewtopic.php?t=8076)
```